consult.el

```diff
@@ -3530,6 +3530,13 @@ If no MODES are specified, use currently active major and minor modes."
 
 ;;;;; Command: consult-yank
 
+(defcustom consult-yank-from-kill-ring-rotate
+  (if (symbolp yank-from-kill-ring-rotate) yank-from-kill-ring-rotate t)
```

Comment:
I'd suggest `(bound-and-true-p yank-from-kill-ring-rotate)` as the default value.

Reply:
But I still have a suggestion: use `boundp` instead of `symbolp`.